input/pagecontent/index.md

```diff
@@ -34,3 +34,20 @@ Note that:
 * [Log issues](https://jira.hl7.org/secure/CreateIssue.jspa), [submit a PR!](https://github.com/fhircast/docs)
 * [Converse at chat.fhir.org](https://chat.fhir.org/#narrow/stream/179271-FHIRcast)
 
+### FHIR Publication Details

```

Comment:
This is largely good, informative content. It's at odds with the tone of the rest of the page, which is a gentle introduction to the functional domain of the specification. 

How about creating a page for this content or identifying another appropriate page?

http://build.fhir.org/ig/HL7/fhircast-docs/branches/addUnsubscribeResponse/#cross-version-analysis

Reply:
Discussed on today's FHIRcast. We'll move the Pub Details into the Change Log page, and rename Change Log to Pub Details.